src/Symfony/Bundle/TwigBundle/DependencyInjection/Compiler/ExtensionPass.php

```diff
@@ -71,5 +71,17 @@ public function process(ContainerBuilder $container)
         if ($container->has('assets.packages')) {
             $container->getDefinition('twig.extension.assets')->addTag('twig.extension');
         }
+
+        if (class_exists('Symfony\Component\Yaml\Parser')) {
```

Comment:
As the definition of the container now depends on the existence of a class, we should define a ``ClassExistenceResource`` and add such a resource to the container for such patterns (outside the if), so that the debug container is reloaded if you install the class